source/_docs/configuration/events.markdown

```diff
@@ -57,6 +57,18 @@ Event `scene_reloaded` is fired when scenes have been reloaded and thus might ha
 
 This event has no additional data.
 
+## Event `event_mqtt_reloaded`
+
+Event `event_mqtt_reloaded` is fired when Manually configured MQTT entities have been reloaded and entities thus might have changed.
+
+This event has no additional data.
+
+## Event `event_template_reloaded`
+
+Event `event_template_reloaded` is fired when Template entities have been reloaded and entities thus might have changed.
+
+This event has no additional data.
```

Comment:
These are integration invents, I would recommend documentation these things with the integration pages instead.

Reply:
right. I thought we needed to have it here so all HA core generated events would be together when clicking the Core Objects Events link in the documentation.

Let me see if I can manage and move this ;-)

is that also why the test fails?

Reply:
> is that also why the Jekyll test fails?

No that is unrelated and work in progress.

Reply:
ok. to be sure, you want me to add these to https://www.home-assistant.io/integrations/template and https://www.home-assistant.io/integrations/mqtt ?

Since the Template page is quite long, would you have a preferable spot for this? I could take it all the way down, risking it will never be found.  the other logical spot would be just above https://www.home-assistant.io/integrations/template#state-based-template-sensors but that might be too prominent?